build/localnet/Tiltfile

```diff
@@ -3,25 +3,29 @@ load("ext://helm_resource", "helm_resource", "helm_repo")
 load("ext://namespace", "namespace_create")
 load("ext://restart_process", "docker_build_with_restart")
 
+tiltfile_dir = os.path.dirname(config.main_dir)
```

Comment:
Thank you for doing this! I think it's going to make the codebase much easier to understand